### PR TITLE
Add RUM integration for Cypress

### PR DESCRIPTION
**Pull request by milena-markovic**
### What does this PR do?

Adds Datadog RUM integration tile for Cypress apps.

+Reorder previously merged rum integrations in the codeowners file

### Motivation

Surfacing the existing Cypress-RUM integration

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

- [x] Logo alias ready in druids

### CODE REVIEW

**Comment by hestonhoffman** [comment]
I've added an [editorial review card for the docs team](https://datadoghq.atlassian.net/browse/DOCS-3719)

**Comment by milena-markovic** [comment]
Hi @hestonhoffman I should've mentioned this in the PR description, but all of the content for this batch of RUM integrations have been pre-reviewed and edited by @alai97 who I've already pinged for approval, so not sure if we need tickets for them

**Review by hithwen — APPROVED** [review]

**Comment by hestonhoffman** [comment]
@milena-markovic, no problem! Austin is out for the week, but I'll take a look at these today

File: rum_cypress/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Looking at test results with distributed tracing details to comprehend why your Cypress tests are slow     
+- Troubleshooting performance gaps in your end-to-end Cypress tests with data collected from RUM & Session Replay
+- Monitoring, capturing, and visually replaying real user sessions
+
+
+## Setup
+
+For more information about integrating Cypress tests with RUM & Session Replay, see the [CI Visibility-RUM integration documentation][2].
+
+Setup
```
Comment (hestonhoffman) [inline rum_cypress/README.md:17]:
```suggestion
```

File: rum_cypress/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Collect RUM events 
+
+To start collecting Real User Monitoring events from your application, see [Cypress Monitoring][3]. 
+
+### Collect traces 
+
+Your application automatically sends traces to Datadog.
+
+Data Collected
```
Comment (hestonhoffman) [inline rum_cypress/README.md:27]:
```suggestion
## Data Collected
```

File: rum_cypress/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Need help? Contact [Datadog Support][5]. 
+
+## Further Reading 
+
+Additional helpful documentation, links, and articles: 
+
+- [CI Visibility][6]
+
+
+# Links
```
Comment (hestonhoffman) [inline rum_cypress/README.md:48]:
```suggestion
```

**Review by hestonhoffman — APPROVED** [review]
LGTM!

**Pull request merged**